### DISCUSSION TITLE
Thank You

### DISCUSSION THREAD

**Discussion by abbasou** [ChurchApps/ChurchAppsSupport#311] [FreeShow] [answered]
First off, I want to say thank you for taking on this project. As the media/tech guy in a relatively small church, I'm always looking for good, affordable alternatives to the expensive professional software that's out there. FreeShow seems to be off to a great start as a solid alternative to our current ProPresenter 7. We've actually not renewed our PP7 license, so we're stuck on an older version with no official support, hoping it continues to run as it has.

So thank you. I wish I could contribute, but alas, all I can do is test a bit and offer a nice long wishlist of feature requests and bugs. :-)

**Comment by vassbo** [comment, marked as answer]
Thank you so much, it means a lot. This is some of the reason I am working on this project! 😄

Bug reports and new feature requests are always wecome. 😊
